Assets/InputActions/PlayerControls.inputactions

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "BaseControls",
+    "maps": [
+        {
+            "name": "Base",
+            "id": "fef48c25-9079-4fbc-b3a4-a89e1504690a",
+            "actions": [
+                {
+                    "name": "Movement",
+                    "type": "Value",
+                    "id": "108ea752-d9ab-4fdb-be7c-a8d2914f4af6",
+                    "expectedControlType": "Vector2",
+                    "processors": "",
+                    "interactions": "",
+                    "initialStateCheck": true
+                }
+            ],
+            "bindings": [
+                {
+                    "name": "2D Vector",
+                    "id": "dcfb76a7-2b91-401d-9123-01903b5341ff",
+                    "path": "2DVector",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Movement",
+                    "isComposite": true,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "Up",
+                    "id": "d6b0235c-2af3-4be3-96b8-2a35edb0d40a",
+                    "path": "<Keyboard>/w",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Movement",
+                    "isComposite": false,
+                    "isPartOfComposite": true
+                },
+                {
+                    "name": "Down",
+                    "id": "7c68d466-de5f-4e19-a665-141772ab58cf",
+                    "path": "<Keyboard>/s",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Movement",
+                    "isComposite": false,
+                    "isPartOfComposite": true
+                },
```

Comment:
you can remove the down binding, i dont belive it will be necessary for the game